src/components/delegate-monitor/delegate-monitor.html

```diff
@@ -345,9 +343,7 @@ <h1>Delegate Monitor</h1>
 								<tr data-ng-repeat='d in vm.standbyDelegates | orderBy:$root.natural(vm.tables.standby.predicate):vm.tables.standby.reverse track by d.rate'>
 									<td class="left-padding-xs left-padding-s left-padding-m left-padding-l double">{{d.rate}}</td>
 									<td class="right-padding-xs">
-										<a data-ng-if="d.proposal" class="glyphicon glyphicon-user proposal-icon" data-ng-href="https://forum.lisk.io/viewtopic.php?f=48&t={{d.proposal.topic}}"
-											title="{{d.proposal.description}}" target="_new"></a>&nbsp;
-										<a data-account-href="d" data-type="delegate">{{d.username}}</a>
+										<a data-account-href="d" class="glyphicon glyphicon-user" data-type="delegate">{{d.username}}</a>
```

Comment:
This should be `<a data-account-href="d" data-type="delegate">{{d.username}}</a>`